grafana/mattermost-performance-monitoring-v2.json

```diff
@@ -2608,12 +2858,13 @@
     ]
   },
   "time": {
-    "from": "now-6h",
```

Comment:
Not a big deal but just want to confirm this is wanted and not just a result of exporting the current state.

Reply:
This wasn't intentional